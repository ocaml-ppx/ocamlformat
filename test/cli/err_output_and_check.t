  $ echo 'let x = 1' > a.ml

  $ ocamlformat --output x.ml --check a.ml
  ocamlformat: Cannot specify --output with --check
  [1]
