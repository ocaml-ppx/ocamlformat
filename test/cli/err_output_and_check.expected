ocamlformat: Cannot specify --output with --check
